### ISSUE TITLE
themeltspice.sh -l throws error

### ISSUE THREAD

**Issue by hb020** [johan162/ltspice-osx-themes#1]
MacOS 12.4

Fresh install of themeltspice via brew

Bad:
```
% themeltspice.sh -l
/usr/local/bin/themeltspice.sh: line 482: theme_name: unbound variable

% themeltspice.sh
/usr/local/bin/themeltspice.sh: line 495: theme_name: unbound variable

% themeltspice.sh -V
/usr/local/bin/themeltspice.sh: line 495: theme_name: unbound variable
```

OK:
```
% themeltspice.sh -v
v1.5.0

% themeltspice.sh -l default
Theme 'default' exists in '/Users/{redacted}/.ltspice_themes/themes.ltt'
```

When changing line 428 from
`declare theme_name`
to
`declare theme_name=""`
it is better.

**Comment by johan162** [comment]
I'm afraid I cannot reproduce this error with the default bash in OSX 12.4 (and I only verify with the default version)

**Intel**
```
% bash --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin21)
Copyright (C) 2007 Free Software Foundation, Inc.
```

**ARM**
```
% bash --version
GNU bash, version 3.2.57(1)-release (arm64-apple-darwin21)
Copyright (C) 2007 Free Software Foundation, Inc.
```

The default bash is quite old (yes, I know ...) and newer versions of bash have improved on error checking. Can you confirm the bash version?

Anyway,  I will consider updating as per your suggestion once I confirmed the cause of the error if it makes it more compatible with newer versions of bash.

ADDENDUM: This is the results I get

```
% themeltspice.sh -v
v1.5.0

% themeltspice.sh -l
Listing themes in '/Users/ljp/.ltspice_themes/themes.ltt'
 1. default
 2. sakabug
 3. twilight-after-dawn
 4. dracula
 5. softdark
 6. blackwhite
 7. redblack
 8. bbking
```
and

```
% themeltspice.sh -v
v1.5.0

% themeltspice.sh
*** ERROR *** No theme name specified.
Set or create a named color theme for LTSpice
Usage:
%themeltspice.sh [-f <FILE>] [-d] [-l] [-h] [-p] [-q] [-v] [-x <THEME>] [-y] [<THEME>]
-d          : Dump current plist to default or named theme file as specified theme
-f <FILE>   : Use the specified file as theme file
-h          : Print help and exit
-l [<NAME>] : List themes in default or named theme file or íf <NAME> is specified check if <NAME> theme exists
-p          : List content in LTSpice plist file
-q          : Quiet  (no output to stdout)
-V          : Verbose status output
-v          : Print version and exit
-x <NAME>   : Delete theme NAME from themes file
-y          : Force 'yes' answer to any interactive questions (e.g. deleting theme)
```

**Comment by hb020** [comment]
In my case, default shell is zsh, but both my zsh and bash come from brew, and both freak out.

```
 % zsh --version
zsh 5.9 (x86_64-apple-darwin21.3.0)

% bash --version
GNU bash, version 5.1.16(1)-release (x86_64-apple-darwin21.1.0)
Copyright (C) 2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

% bash
bash-5.1$ themeltspice.sh -l
/usr/local/bin/themeltspice.sh: line 482: theme_name: unbound variable
```

**Comment by johan162** [comment]
As I expected you are not using the standard OSX `bash` but a **much** newer version.

(This might make you run into issues with other programs/utilities that rely on the version of bash that is shipped with OSX 12.4)

Anyway, Thanks for the report and I will push out an updated version today.

**Comment by johan162** [comment]
Fixed in 1.5.1
